hardhat/tasks/sips/args/sipArgs.js

```diff
@@ -1104,6 +1104,159 @@ const getArgsSIP0077 = async (hre) => {
     return { args, governor: "GovernorAdmin" };
 };
 
+const getArgsSip0080 = async (hre) => {
+    const {
+        ethers,
+        deployments: { get, log },
+    } = hre;
+
+    const abiCoder = new ethers.utils.AbiCoder();
+    const priceFeeds = await ethers.getContract("PriceFeeds");
+    const priceFeedsOwner = await priceFeeds.owner();
+
+    const wrbtcAddress = (await get("WRBTC")).address;
+    const priceFeedMocAddress = (await get("PriceFeedsMoc")).address;
+
+    const args = {
+        targets: [priceFeeds.address],
+        targetOwnerValidationAddresses: [priceFeedsOwner],
+        values: [0],
+        signatures: ["setPriceFeed(address[],address[])"],
+        data: [
+            abiCoder.encode(["address[]", "address[]"], [[wrbtcAddress], [priceFeedMocAddress]]),
+        ],
+        description:
+            // @todo update description
+            "SIP-0080: Update MoCPriceFeeds, Details: https://github.com/DistributedCollective/SIPS/blob/8cb4f72/SIP-0080.md, sha256: ",
+    };
+    return { args, governor: "GovernorAdmin" };
+};
+
+const getArgsSip0081 = async (hre) => {
```

Comment:
1) pls assign the SIP-0085 
```suggestion
const getArgsSip0085 = async (hre) => {
```
also rename the file and the Description text  

2) the SIP-0085 (zero pricefeeds upgrade) should do the following:
   1) replace logic of the Zero PriceFeed upgradeable proxy with the PriceFeedRevertOnStalePrice contract
   2) call priceFeed.setAddresses(`mainPriceFeed(MoC oracle)`, backupPriceFeed(DummyFallbackOracle))

Reply:
Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/549/commits/d47e661d64be04ec9d03dbffbe6d7b0dd835ca50)